lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/FileBasedSystemAccessControl.java

```diff
@@ -188,6 +194,9 @@ private FileBasedSystemAccessControl(
     public static class Factory
             implements SystemAccessControlFactory
     {
+        private static final String HTTP_CLIENT_NAME = "system-security";
```

Comment:
Maybe `access-control`? Then we could same name in connector access control too.